### PR TITLE
start es, kibana, postgres from docker-compose

### PR DESCRIPTION
**Pull request by nrzhao4**
- for [SEAB-1948](https://ucsc-cgl.atlassian.net/browse/SEAB-1948)
- running `docker-compose up -d` brings up elasticsearch and postgres containers. kibana instance can be uncommented if needed

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    # container_name: kib01
+    # ports:
+      # - 5601:5601
+    # environment:
+      # ELASTICSEARCH_URL: http://es01:9200
+      # ELASTICSEARCH_HOSTS: http://es01:9200
+    # networks:
+      # - elastic
+  postgres_db:
+    image: postgres:11.6
```
Comment (garyluu) [inline docker-compose.yml:41]: @coverbeck 

11.8 or something?

Comment (coverbeck) [reply docker-compose.yml:41]: Yeah, we're apparently on 11.8 in prod -- PG auto updates.

**Review by coverbeck — COMMENTED** [review]
Is there any way, and the answer is probably no, to use the settings from compose_setup? So that we don't run into the issue again where during the last upgrade it ran for you locally but failed in dev, because dev had different settings.

This will be hard because:

1. AFAIK, you can't embed a file in a docker-compose file
2. The settings in compose_setup are for a prod instance, some of which may not be applicable for our local boxes.
3. It's a template file in compose_setup, so some settings might have to be filled in to be usable.

So, you'd probably have to write a script to do it, and it may not be worth the effort.

Also, if we move to ES as a service, then there won't even be a template in compose_setup

Still, can you please think about it for 5 minutes and see if there's some sort of reasonable solution?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+version: '2.2'
```
Comment (denis-yuen) [inline docker-compose.yml:1]: Could use some comments for new developers to know how this is different from the compose files in compose setup.

**Comment by nrzhao4** [comment]
> Is there any way, and the answer is probably no, to use the settings from compose_setup? So that we don't run into the issue again where during the last upgrade it ran for you locally but failed in dev, because dev had different settings.
> 
> This will be hard because:
> 
> 1. AFAIK, you can't embed a file in a docker-compose file
> 2. The settings in compose_setup are for a prod instance, some of which may not be applicable for our local boxes.
> 3. It's a template file in compose_setup, so some settings might have to be filled in to be usable.
> 
> So, you'd probably have to write a script to do it, and it may not be worth the effort.
> 
> Also, if we move to ES as a service, then there won't even be a template in compose_setup
> 
> Still, can you please think about it for 5 minutes and see if there's some sort of reasonable solution?

I think the easiest solution for now is to figure out which settings can be the same, and have the two be similar as possible

**Comment by nrzhao4** [comment]
> > Is there any way, and the answer is probably no, to use the settings from compose_setup? So that we don't run into the issue again where during the last upgrade it ran for you locally but failed in dev, because dev had different settings.
> > This will be hard because:
> > 
> > 1. AFAIK, you can't embed a file in a docker-compose file
> > 2. The settings in compose_setup are for a prod instance, some of which may not be applicable for our local boxes.
> > 3. It's a template file in compose_setup, so some settings might have to be filled in to be usable.
> > 
> > So, you'd probably have to write a script to do it, and it may not be worth the effort.
> > Also, if we move to ES as a service, then there won't even be a template in compose_setup
> > Still, can you please think about it for 5 minutes and see if there's some sort of reasonable solution?
> 
> I think the easiest solution for now is to figure out which settings can be the same, and have the two be similar as possible

ES here and in compose_setup have pretty similar settings now. Added a change to the compose_setup as part of [this PR](https://github.com/dockstore/compose_setup/pull/171). One difference is in compose_setup, the compose file references elasticsearch.yml for some settings, in this compose file all the settings are under `environment`

**Review by denis-yuen — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

**Pull request merged**